cpp/include/cpu_operations.h

```diff
@@ -156,7 +156,28 @@ class CpuOperations {
       return a.inverse();
     }
   }
-static Vector<T> Norm(const Matrix<T> &a,
+/// static Vector <T> Norm( const Matrix <T> &a,
+/// const int &p = 2, const int &axis = 0) calculates the norm of
+/// the values in an m x n dependent of the input p and axis.
+/// The norm is returned in the form of a vector. If the axis is 0, 
+/// the norm will be calulated column wise and the size of the 
+/// output vector will be dependent on n. If the axis is 1, the
+/// norm will be calculated row-wise and the size of the vector
+/// will be dependent on m.  
+///
+/// \param a
+/// const Matrix <T> &a
+/// \param b
+/// \const int &p 
+/// \param c
+/// \const int &axis
+///
+/// \return 
+/// Vector <T>
+/// \sa 
+/// \ref Frobenius Norm : Frobenius Norm is similar to Norm where 
```

Comment:
Also here the \ref doesn't give me link to Frobenius Norm. Could you make a link? Try 
\ref FrobeniusNorm or something. You could discuss with Leslie on how to make this work.